.github/workflows/book.yml

```diff
@@ -0,0 +1,32 @@
+name: halo2 book
+
+on:
+  push:
+    branches:
+      - master
+      - next
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          # Pinned until https://github.com/rust-lang/rust/issues/93476 is fixed.
+          toolchain: nightly-2022-01-18
```

Comment:
Seems like this issue has been fixed already.

Reply:
fixed